docker/test/integration/runner/compose/docker_compose_nginx.yml

```diff
@@ -0,0 +1,11 @@
+version: '2.3'
+services:
+    # nginx server to host static files.
+    # Accepts only PUT data by test.com/path and GET already existing data on test.com/path.
+    # Files will be put into /usr/share/nginx/files.
+
+    nginx:
+        image: kssenii/nginx-test:1.1
```

Comment:
How can I look at `Dockerfile` for this image?
Maybe it worth to move this image into `clickhouse` namespace?

Reply:
And by the way, maybe official nginx image + file overrides will be enough?

Reply:
> How can I look at Dockerfile for this image?

Unfortunately I do not have it anymore.

> And by the way, maybe official nginx image + file overrides will be enough?

As I remember I tried it when adding these tests, but something did not work and I decided to write my own image, but I do not remember what exactly, it was more than a year ago...

Reply:
We have to write new Dockerfile then, put it into ClickHouse repo and upload image to `clickhouse` namespace

Reply:
Created an issue: https://github.com/ClickHouse/ClickHouse/issues/43182